docs/api/IoT.js-API-BLE.md

```diff
@@ -9,64 +9,42 @@ The following shows ble module APIs available for each platform.
 | ble.setServices | O | O | X |
 
 
-### Contents
+# BLE - Bluetooth Low Energy
 
-- [BLE](#ble)
-    - [Module Functions](#module-functions)
-        - [`ble.startAdvertising(name, serviceUuids[, callback])`](#blestartadvertisingname-serviceuuids-callback)
-        - [`ble.stopAdvertising(callback)`](#blestopadvertisingcallback)
-        - [`ble.stopAdvertising(callback)`](#blestopadvertisingcallback-1)
-        - [`ble.setServices(services[, callback])`](#blesetservicesservices-callback)
-    - [Events](#events)
-        - [`'stateChange'`](#statechange)
-        - [`'advertisingStart'`](#advertisingstart)
-
-
-# BLE
-
-
-## Module Functions
-
-
-### `ble.startAdvertising(name, serviceUuids[, callback])`
-* `name <String>` - maximum 26 bytes
-* `serviceUuids <Array>`
+### ble.startAdvertising(name, serviceUuids[, callback])
+* `name` {string} - maximum 26 bytes
+* `serviceUuids` {Array}
   * 1 128-bit service UUID
   * 1 128-bit service UUID + 2 16-bit service UUIDs
   * 7 16-bit service UUID
-* `callback <Function(error)>`
+* `callback` {Function}
+  * error {Error}
 
 Starts advertising.
 
-ble.state must be poweredOn before advertising is started. 
-ble.on('stateChange', callback(state)); can be used register for state change events.
+`ble.state` must be poweredOn before advertising is started.
+`ble.on('stateChange', callback(state));` can be used to register for state change events.
 
 
-### `ble.stopAdvertising(callback)`
-* `callback <Function(error)>`
+### ble.stopAdvertising(callback)
+* `callback` {Function}
+  * error {Error}
```

Comment:
I also added explanations for callbacks in the GPIO module. Mostly the same text with little modifications, you can copy them here.